packages/camera/camera_windows/windows/CMakeLists.txt

```diff
@@ -0,0 +1,90 @@
+cmake_minimum_required(VERSION 3.15)
+set(PROJECT_NAME "camera_windows")
+project(${PROJECT_NAME} LANGUAGES CXX)
+
+# This value is used when generating builds using this plugin, so it must
+# not be changed
+set(PLUGIN_NAME "${PROJECT_NAME}_plugin")
+
+list(APPEND PLUGIN_SOURCES
+  "camera_plugin.h"
+  "camera_plugin.cpp"
+  "camera.h"
+  "camera.cpp"
+  "capture_controller.h"
+  "capture_controller.cpp"
+  "capture_controller_listener.h"
+  "capture_engine_listener.h"
+  "capture_engine_listener.cpp"
+  "string_utils.h"
+  "string_utils.cpp"
+  "device_info.h"
+  "device_info.cpp"
+)
+
+add_library(${PLUGIN_NAME} SHARED
+  "camera_windows.cpp"
+  "include/camera_windows/camera_windows.h"
+  ${PLUGIN_SOURCES}
+)
+
+apply_standard_settings(${PLUGIN_NAME})
+set_target_properties(${PLUGIN_NAME} PROPERTIES
+  CXX_VISIBILITY_PRESET hidden)
+target_compile_definitions(${PLUGIN_NAME} PRIVATE FLUTTER_PLUGIN_IMPL)
+target_include_directories(${PLUGIN_NAME} INTERFACE
+  "${CMAKE_CURRENT_SOURCE_DIR}/include")
+target_link_libraries(${PLUGIN_NAME} PRIVATE flutter flutter_wrapper_plugin)
+target_link_libraries(${PLUGIN_NAME} PRIVATE mf mfplat mfreadwrite mfuuid d3d11)
```

Comment:
Do these limit the version of Windows that is supported? If so, let's mention that in the README.

Reply:
This is good point.
These libraries are available from Windows 7 [Media Foundation Headers and Libraries](https://docs.microsoft.com/en-us/windows/win32/medfound/media-foundation-headers-and-libraries)
[Minimum supported client for used IMFCaptureEngine](https://docs.microsoft.com/en-us/windows/win32/api/mfcaptureengine/nn-mfcaptureengine-imfcaptureengine) is Windows 8
I have tested camera plugin with Windows 10 and with Windows 11.
This should be tested with Win8 and Win7 as well, but I haven't been able to do so yet.